ranger-zookeeper/src/main/java/com/flipkart/ranger/zookeeper/ServiceFinderBuilders.java

```diff
@@ -1,35 +1,35 @@
-/**
+/*
  * Copyright 2015 Flipkart Internet Pvt. Ltd.
- *
+ * <p>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
+ * <p>
  * http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package com.flipkart.ranger.zookeeper;
 
-import com.flipkart.ranger.zookeeper.zk.ZkSimpleShardedServiceFinderBuilder;
-import com.flipkart.ranger.zookeeper.zk.ZkUnshardedFinderBuilder;
+import com.flipkart.ranger.core.model.Criteria;
+import com.flipkart.ranger.zookeeper.servicefinder.ZkSimpleShardedServiceFinderBuilder;
+import com.flipkart.ranger.zookeeper.servicefinder.ZkSimpleUnshardedServiceFinderBuilder;
```

Comment:
Remove constructor below and put `@UtilityClass`

Reply:
Part of the lombok cleanup